examples/website/plot/plot-layer/axes-layer.ts

```diff
@@ -222,29 +236,29 @@ export default class AxesLayer extends Layer {
       0, -1, 0, 0, -1, 0
     ];
 
-    const grids = new Model(gl, {
+    const grids = new Model(gl.device, {
       id: `${this.props.id}-grids`,
       vs: gridVertex,
       fs: fragmentShader,
+      bufferLayout: this.getAttributeManager().getBufferLayouts(),
       geometry: new Geometry({
         topology: 'line-list',
         attributes: {
-          positions: new Float32Array(gridPositions),
-          normals: new Float32Array(gridNormals)
+          positions: {value: new Float32Array(gridPositions), size: 3},
+          normals: {value: new Float32Array(gridNormals), size: 3}
         },
         vertexCount: gridPositions.length / 3
-      }),
-      isInstanced: true
+      })
```

Comment:
This works, and types say `isInstanced` no longer exists, but is some indication of instancing expected at this point? `state.instanceCount` is not readily available but could be derived if necessary.

Reply:
@ibgreen is this safe to remove? I see we still have it in `ScatterplotLayer` and other layers